### PR TITLE
Center the map on the tap location when zooming

### PR DESCRIPTION
**Pull request by Mkohm**
Please merge this one before #17 as #16 is based on this one..

### CODE REVIEW

File: lib/flutter_map_tappable_polyline.dart [hunk 1]
```diff
@@ -63,14 +63,14 @@ class TaggedPolyline extends Polyline {
...
-            colorsStop: colorsStop,
-            isDotted: isDotted);
+      points: points,
+      strokeWidth: strokeWidth,
+      color: color,
+      borderStrokeWidth: borderStrokeWidth,
+      borderColor: borderColor,
+      gradientColors: gradientColors,
+      colorsStop: colorsStop,
+      isDotted: isDotted);
```
Comment (tuarrep) [inline lib/flutter_map_tappable_polyline.dart:73]: Could you revert this. It's about code readability

**Review by tuarrep — CHANGES_REQUESTED** [review]
Looks great to me. Just an unwanted space change.

File: lib/flutter_map_tappable_polyline.dart (hunk 1 above)
Comment (Mkohm) [reply lib/flutter_map_tappable_polyline.dart:73]: I cannot get dartfmt to format it that way. Using dartfmt inside android studio..

Comment (Mkohm) [reply lib/flutter_map_tappable_polyline.dart:73]: Also, as the package analyzer will check for formatting issues i don't see the issue?

Comment (tuarrep) [reply lib/flutter_map_tappable_polyline.dart]: Push a commit myself to see

Comment (tuarrep) [reply lib/flutter_map_tappable_polyline.dart]: The action was misconfigured. It did not fail if the score was too low.

**Review by tuarrep — APPROVED** [review]

**Pull request merged**